### ISSUE TITLE
librarian can't find you puppet module

### ISSUE THREAD

**Issue by alexkiousis** [sbadia/vagrant-gitlab#9]
On Jessie after git-cloning and installing librarian-puppet and puppet i get:

```
~/vagrant-gitlab$ librarian-puppet install
Unable to find module 'sbadia/gitlab' on https://forgeapi.puppetlabs.com
```

**Comment by sbadia** [comment]
Hi, thanks for this bug-report,
Indeed, the latest release of  `sbadia/puppet-gitlab` still contain the old-style of puppet-forge description (with the `Modulefile` file), we updated this in the git version, but the `0.1.6` version of `sbadia/puppet-gitlab` isn't released yet (with a `metadata.json` in order to work with the Forge v3 api).
I'll manage this, this week-end.
Thanks!

**Comment by sbadia** [comment]
Fixed with sbadia-gitlab 0.2.0 https://forgeapi.puppetlabs.com/v3/files/sbadia-gitlab-0.2.0.tar.gz
